src/shell/shell_cmds.cpp

```diff
@@ -1374,8 +1436,24 @@ void DOS_Shell::CMD_CALL(char * args){
 	this->call=false;
 }
 
-void DOS_Shell::CMD_DATE(char * args) {
-	HELP("DATE");
+void DOS_Shell::CMD_DATE(char *args)
```

Comment:
do we change args?

Reply:
All CMD_xxxx functions use the ```char *args``` argument, and if I changed it to const then the code won't build.

Reply:
Apologies for the bad suggestion, @Wengier . 
Thanks for the explanation 👍

Reply:
You made so many helpful comments and suggestions in this single PR, which I highly appreciate.